@xen-orchestra/web/src/components/backups/configuration/BackupJobsSchedules.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <UiCard class="backup-jobs-schedules">
+    <UiTitle> {{ t('schedules') }} </UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTopBottomTable :selected-items="0" :total-items="0">
+          <UiTablePagination v-bind="paginationBindings" />
+        </UiTopBottomTable>
+      </div>
+      <VtsDataTable is-ready :no-data-message="backupJobs.length === 0 ? t('no-backup-available') : undefined">
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" class="checkbox">
+                <div v-tooltip="t('coming-soon')">
+                  <UiCheckbox disabled accent="brand" />
+                </div>
+              </th>
+              <th v-else-if="column.id === 'more'">
+                <UiButtonIcon v-tooltip="t('coming-soon')" icon="fa:ellipsis" accent="brand" disabled size="small" />
+              </th>
+              <th v-else>
+                <div v-tooltip>
+                  <VtsIcon size="medium" :name="headerIcon[column.id]" />
+                  {{ column.label }}
+                </div>
+              </th>
+            </template>
+          </tr>
+        </template>
+        <template #tbody>
+          <tr v-for="row of backupJobsRecords" :key="row.id" :class="{ selected: selectedBackupJobId === row.id }">
+            <td v-for="column of row.visibleColumns" :key="column.id" :class="{ checkbox: column.id === 'checkbox' }">
+              <div v-if="column.id === 'checkbox'" v-tooltip="t('coming-soon')">
+                <UiCheckbox disabled accent="brand" :value="row.id" />
+              </div>
+              <UiButtonIcon
+                v-else-if="column.id === 'more'"
+                v-tooltip="t('coming-soon')"
+                icon="fa:ellipsis"
+                accent="brand"
+                disabled
+                size="small"
+              />
+              <div v-else-if="column.id === 'schedule'">
+                <UiLink size="medium" icon="object:backup-job" :href="`/#/backup/${column.value.jobId}/edit`">
+                  {{ column.value.name }}
+                </UiLink>
+              </div>
+              <div v-else-if="column.id === 'id'" v-tooltip class="text-ellipsis">
+                {{ column.value }}
+              </div>
+              <div v-else-if="column.id === 'cron-pattern'" v-tooltip class="text-ellipsis">
+                {{ column.value }}
+              </div>
```

Comment:
This can be done after all other checks, and simplified:
```suggestion
              <div v-else v-tooltip class="text-ellipsis">
                {{ column.value }}
              </div>
```